uptane-standard.md

```diff
@@ -859,64 +852,90 @@ In order to perform full verification, an ECU SHALL perform the following steps:
+7. Check that the latest attested time is lower than the expiration timestamp in this Snapshot metadata file. If the new Snapshot metadata file is expired, discard it, abort the update cycle, and report the potential freeze attack. (Checks for a freeze attack.)
+
+#### How to check Targets metadata {#check_targets}
+
+1. Download the number of bytes either specified in the Snapshot metadata file, or some Z number of bytes, constructing the metadata filename as defined in {{metadata_filename_rules}}. The value for Z is set by the implementor. For example, Z may be tens of kilobytes.
+2. The hashes (if any), and version number of the new Targets metadata file MUST match the latest Snapshot metadata. If the new Targets metadata file does not match, discard it, abort the update cycle, and report the failure. (Checks for a mix-and-match attack.) Skip this step if checking Targets metadata on a partial-verification ECU; partial-verification ECUs will not have Snapshot metadata.
+3. Check that it has been signed by the threshold of keys specified in the relevant metadata file (Checks for an arbitrary software attack):
+    1. If checking top-level targets metadata, the threshold of keys is specified in the Root metadata.
+    2. If checking delegated targets metadata, the threshold of keys is specified in the targets metadata file that delegated authority to this role.
+4. Check that the version number of the previous Targets metadata file, if any, is less than or equal to the version number of this Targets metadata file. (Checks for a rollback attack.)
+5. If this Targets metadata file indicates that the Timeserver key should be rotated, then reset the clock used to determine the expiration of metadata to a minimal value (e.g. zero, or any time that is guaranteed to not be in the future based on other evidence). It will be updated in the next cycle.
+6. Check that the latest attested time is lower than the expiration timestamp in this Targets metadata file. (Checks for a freeze attack.)
+7. If checking targets metadata from the Director repository, verify that there are no delegations.
+8. If checking targets metadata from the Director repository, check that no ECU identifier is represented more than once.
+
+#### How to resolve delegations {#resolve_delegations}
+
+To properly check targets metadata for an image, an ECU MUST locate the metadata file(s) for the role (or roles) that have the authority to sign the image. This metadata might be located in the top-level targets metadata, but it also may be delegated to another role--or to multiple roles. Therefore, all delegations MUST be resolved using the following recursive procedure, beginning with the top-level targets metadata file. (Note: "Stack" here is used as defined in RFC TODO.)
```

Comment:
What is the RFC TODO?

Reply:
Oops, that made it through accidentally. I was debating whether it was necessary to explicitly define what is meant by "stack", and I thought perhaps there was an RFC with a definition that I could just reference. I was on a plane at the time, though, so I didn't have the chance to look it up. After a bit of searching now, I can't find an RFC to cite.

On second reflection, though, I think that a stack is enough of a well-understood concept that we don't really need to explicitly or normatively define it. What do you think?